entrypoint.sh

```diff
@@ -315,8 +315,8 @@ then
             k get ns "${INPUT_NAMESPACE}" || echo_exit "Namespace doesn't exist"
 
             echo "-- Get release list"
-            release=$(helm list -a -q -n "${INPUT_NAMESPACE}" | grep "${INPUT_RELEASE_NAME}" || true)
-            if [ -n "${release}" ]
+            release=$(helm list -a -q -n "${INPUT_NAMESPACE}" | grep "${INPUT_RELEASE_NAME}" || echo "missing")
+            if [ "${release}" = "missing" ]
```

Comment:
I think this is a better pattern for what's intended here. We skip the confusing middle man and directly check the results of the `helm list | grep`.

```suggestion
            if helm list -a -q -n "${INPUT_NAMESPACE}" | grep "${INPUT_RELEASE_NAME}"
```